src/utils/file/file.spec.ts

```diff
@@ -91,7 +91,26 @@ describe('FileService', () => {
                     createMockFile('valid.jpeg'),
                     createMockFile('valid.pdf'),
                     createMockFile('valid.mp4'),
-                    createMockFile('invalid')
+                    createMockFile('invalid.mov')
+                ])
+            );
+
+            expectValidationResults(5, MFileRejectionCause.FILE_TYPE);
+        });
+
+        it('should reject files with rejectedExtensions', () => {
```

Comment:
Le test des extensions rejetées
@Mboulianne

Reply:
les tests testent trop de chose en même temps : je suggérerais : 
3 tests où on met seulement une valeur dans les acceptedExtention (qui peut être n'importe quoi pas besoin d'être une vraie extension, genre 'bla' ou 'extention') : un test doit ensuite valider une liste où il y a seulement des extensions ok, une autre avec des extension toutes pas ok, et un test avec les deux (ou pourrait possiblement mettre deux extensions dans les accepted pour combiner réduire les tests)
describe('Given deux accepted extension', () => {
describe('When validation files with only accepted extension', () => {
it(shouldBeOK)
})
describe('When validation files with no accepted extension', () => {
it(shouldBeNOTOK)
})
describe('When validation files with some accepted extension', () => {
it(shouldBeNOTOK)
})
})
Ensuite, faire un test pour valider qu'on accepte les extensions peut importe la case
Ensuite, faire la même chose pour les rejected

Reply:
** Il se passe quoi si on est con et qu'on met la même extension dans rejected et dans accepted, possiblement ajouter une validation là dessus quand on fait le set de validationOptions???

Reply:
Si une extensions est présente dans les 2, elle sera refusée.  Bonne idée de rajouter un test la dessus!